index.js

```diff
@@ -0,0 +1,99 @@
+/* eslint-disable no-unused-vars */
+function setup() {
+    const formElem = document.getElementById("createForm")
+    formElem.addEventListener("submit", e => {
+        e.preventDefault()
+        createURL()
+    })
+}
+
+function createURL() {
+    const shorturl = document.getElementById("url").value
+    const name = document.getElementById("name").value
+    const date = getDate()
+    if(!shorturl){
+        document.getElementById("response").innerHTML = "Bitte gib eine URL an"
+        return
+    }
+    if(!shorturl.startsWith("https://") && !shorturl.startsWith("http://")){
+        document.getElementById("response").innerHTML = "Bitte gib eine gültige URL an"
+        return
+    }
+    var params = JSON.stringify({url: shorturl, name, date})
+    var xhr = new XMLHttpRequest()
+    xhr.onreadystatechange = function() {
+        if (this.readyState == 4) document.getElementById("response").innerHTML = this.responseText
+    }
+    xhr.open("POST", "https://api.tomatenkuchen.eu/short", true)
+    xhr.setRequestHeader("Content-Type", "application/json")
+
+    xhr.send(params)
+}
+
+function copy() {
+      navigator.clipboard.writeText(document.getElementById("resulturl").href).catch(err => {
+        alert("Link konnte nicht kopiert werden")
+        console.error("Link konnte nicht kopiert werden", err)
+    })
+}
+
+function update(){
+    let select = document.getElementById('date');
+	let option = select.options[select.selectedIndex];
+    switch (option.value) {
+        case "userdef": document.getElementById("date_userdef").type = "datetime-local"; 
+            var dateElem = document.getElementById("date_userdef")
+            dateElem.min = dateAdd(new Date, "minute", 2).toISOString().slice(0, -8)
+            dateElem.value = dateAdd(new Date, "minute", 2).toISOString().slice(0, -8)
+            break;
+        default: document.getElementById("date_userdef").type = "hidden"; 
+            break;
+    }
+}
+
+function dateAdd(date, interval, units) {
+    if(!(date instanceof Date))
```

Comment:
```suggestion
    if (!(date instanceof Date))
```

Reply:
Applies to all function names and if's, so `function update(){` -> `function update() {`